operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -137,6 +137,46 @@ local filename = 'mimir-writes.json';
         )
       )
     )
+    .addRowIf(
+      $._config.forwarding_enabled,
+      $.row('Distributor Forwarding')
+      .addPanel(
+        $.panel('Requests / sec') +
+        $.successFailurePanel(
+          |||
+            sum(  
+              rate(
+                cortex_distributor_forward_requests_total{%(distributorMatcher)s}[$__rate_interval]
+              )
+            )
+            - 
+            sum(
+              rate(
+                cortex_distributor_forward_errors_total{%(distributorMatcher)s}[$__rate_interval]
+              )
+            )
+          ||| % [
```

Comment:
Given you're using named placeholders, you should leverage on that:

```
{ distributorMatcher: $.jobMatcher($._config.job_names.distributor) }
``` 

Reply:
ah, thanks: 001ef5947a